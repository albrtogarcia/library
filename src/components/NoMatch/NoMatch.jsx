import './NoMatch.scss'

export const NoMatch = () => (
  <div className="nomatch">
    <h2>404 - Not found</h2>
  </div>
)
