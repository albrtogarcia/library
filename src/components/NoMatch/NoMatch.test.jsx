import { describe, expect, it } from 'vitest'
import { NoMatch } from './NoMatch.jsx'

describe('NoMatch', () => {
  it('matches snapshot', () => {
    const element = NoMatch()
    expect(element).toMatchSnapshot()
  })
})
