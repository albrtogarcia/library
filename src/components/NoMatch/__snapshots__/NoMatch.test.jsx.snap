// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NoMatch > matches snapshot 1`] = `
<div
  className="nomatch"
>
  <h2>
    404 - Not found
  </h2>
</div>
`;
